dev-packages/application-manager/src/generator/backend-generator.ts

```diff
@@ -192,6 +192,8 @@ serverAddress.then((addressInfo) => {
 });
 
 globalThis.serverAddress = serverAddress;
+globalThis.extensionInfo = ${this.prettyStringify(this.pck.extensionPackages.map(({ name, version }) => ({ name, version }))) };
```

Comment:
Suggestion: Since the information is statically available, (unlike the `serverAddress`), we should set the `extensionInfo` value before importing from `./server`, as adopters might want to access this information during load time.